src/video/uikit/SDL_uikitwindow.m

```diff
@@ -85,7 +87,13 @@ static int SetupWindowData(SDL_VideoDevice *_this, SDL_Window *window, UIWindow
     SDL_UIKitDisplayData *displaydata = (__bridge SDL_UIKitDisplayData *)display->driverdata;
     SDL_uikitview *view;
 
-    CGRect frame = UIKit_ComputeViewFrame(window, displaydata.uiscreen);
+    CGRect frame =
+#if !TARGET_OS_XR
```

Comment:
`#if TARGET_OS_XR`...
Why didn't you go back and change these after the first round of feedback?

Reply:
Seems I missed a few. Went back and fixed them.